### PR TITLE
Some translation slips.

### PR DESCRIPTION
**Pull request by easyMaxi**
Not perfect, but I think better than before.

Made some language specific changes.
Except "keys" I did not touch technical terms.

### CODE REVIEW

**Comment by dadav** [comment]
This will have no effect if you just change the po file. :)

**Comment by dadav** [comment]
> 
> 
> Not perfect, but I think better than before.
> 
> Made some language specific changes.
> Except "keys" I did not touch technical terms.

So, to make this work, you have to compile the po file to an mo file! Could you do that?

File: pwnagotchi/locale/de/LC_MESSAGES/voice.po [hunk 1]
```diff
@@ -203,11 +203,11 @@ msgstr "Ops, da ist was schief gelaufen... Starte neu..."
 
 #, python-brace-format
 msgid "Kicked {num} stations\n"
-msgstr "{num} Stationen gekicked\n"
+msgstr "{num} Stationen gekickt\n"
 
 #, python-brace-format
 msgid "Made {num} new friends\n"
-msgstr "{num} Freunde gefunden\n"
+msgstr "{num} neue Freunde gefunden\n"
```
Comment (WheresAlice) [inline pwnagotchi/locale/de/LC_MESSAGES/voice.po]: should `Freunde` be capitalised like it is here?

Comment (noiob) [reply pwnagotchi/locale/de/LC_MESSAGES/voice.po]: yes, any nouns get capitalized in German, not just proper nouns

**Comment by soebbing** [comment]
Hi @easyMaxi! If you're struggling with compiling the `.mo` file, I compiled it already since I wanted to try your changes. You can find the file here: https://github.com/soebbing/pwnagotchi/commit/b1d61d95e60b76df651e7ab23e540a32b72e84bf#diff-7a5ec11fb0fc56cb4e1520722d8c3470 - feel free to cherry pick or just copy it to your PR.

**Comment by easyMaxi** [comment]
Hi @soebbing I guess the easiest way is then that you create a PR out of your (complete) commit. I am currently running the web interface and have no idea how to extend the PR.
I will close this one here.

**Comment by soebbing** [comment]
Hi @easyMaxi! Sure, I will create a new PR with these changes. I thought about it to make the process easier, but I just didn't want to take credit for your changes.

**Comment by easyMaxi** [comment]
It's very nice of you that you thought about that!
But don't worry about that in this case.

**Pull request closed without merging**